tests/test_frame.py

```diff
@@ -3147,3 +3147,22 @@ def test_get() -> None:
         pd.DataFrame,
     )
     check(assert_type(df.get(["z"], default=1), Union[pd.DataFrame, int]), int)
+
+
+def test_info() -> None:
+    df = pd.DataFrame()
+
+    check(assert_type(df.info(verbose=True), None), type(None))
+    check(assert_type(df.info(verbose=None), None), type(None))
```

Comment:
For any of these tests that are testing values of  `True` and `None`, should also add a test testing a value of `False`